### PR TITLE
659 improve documentation about patch file

### PR DESCRIPTION
**Pull request by temmey**
## Basics

- [x] I added a line to [/doc/CHANGELOG.md](/doc/CHANGELOG.md)
- [x] The PR is rebased with current master.
- [x] Details of what you changed are in commit messages.
- [x] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.

## Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added unit tests for my code
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
      (exceptions are documented)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] Examples are well chosen and understandable

### CODE REVIEW

File: doc/backend/05updating_schema_patch.md [hunk 1]
```diff
@@ -17,7 +17,11 @@ This document explains how to update the `schema.patch` file used by diesel.
    You should now have a generated `schema.rs` in the backend src folder.
 
 3. Copy the `schema.rs` file e.g. to `schema_tmp.rs`.
-4. Run `` diff src/schema.rs `src/schema_tmp.rs` -U6 `` in the backend folder and save the result to the `src/schema.patch` file.
+
+Now manually make changes in `schema.rs`, use the current `schema.patch` to apply important changes
```
Comment (markus2330) [inline doc/backend/05updating_schema_patch.md:21]: I am not sure if I understand the sentence.

```suggestion
    Now manually make changes in `schema.rs` and use the current `schema.patch` to apply changes.
```

**Review by markus2330 — APPROVED** [review]
Maybe you can rephrase a bit?

**Comment by markus2330** [comment]
#684 also has changes to this file.

**Comment by temmey** [comment]
If @kitzbergerg wants to add extra information to his pr, that you should utilize the existing `schema.patch` to incorporate any previous modifications. Otherwise, this pr can be closed.

**Comment by kitzbergerg** [comment]
> If @kitzbergerg wants to add extra information to his pr, that you should utilize the existing `schema.patch` to incorporate any previous modifications. Otherwise, this pr can be closed.

From my side there isn't anything to add. Diesel or at least the Rust compilation will error anyways if you make a mistake while changing the file.

**Pull request closed without merging**